src/transformers/models/layoutlmv2/modeling_layoutlmv2.py

```diff
@@ -855,10 +855,10 @@ def forward(
 
         device = input_ids.device if input_ids is not None else inputs_embeds.device
 
-        visual_shape = list(input_shape)
+        visual_shape = list(torch.empty(size=input_shape).size())
```

Comment:
Seems a bit overkill to initialise a new tensor just to copy the size. If there is a better solution, please let me know.

Reply:
Leveraging `input_ids.size()` doesn't work?

Reply:
Oh yea it does actually, nice! I'll change the PR a bit to avoid the extra allocation